r2/r2/controllers/front.py

```diff
@@ -514,7 +514,7 @@ def GET_newreddit(self, name):
 
     @pagecache_policy(PAGECACHE_POLICY.LOGGEDIN_AND_LOGGEDOUT)
     @require_oauth2_scope("modconfig")
-    @api_doc(api_section.moderation, uses_site=True)
+    @api_doc(api_section.subreddits, uses_site=True)
```

Comment:
:no_bell: I'm not really sure which things should go under moderation and which under subreddits, but as far as minimal changes go, this one makes sense.